lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -11,6 +11,13 @@ def message
 
       private
 
+      def relative_to?(directory)
```

Comment:
I think that we'll have to update the `IssuePathExistenceValidation` to address symlinks and then we should be safe to use `Pathname#realpath` here.